### ISSUE TITLE
Table of contents

### ISSUE THREAD

**Issue by tshemsedinov** [HowProgrammingWorks/CodeStyle#4]
Here is short table of contents to start discussion an editing:
https://github.com/HowProgrammingWorks/CodeStyle/blob/master/JavaScript/README.ru.md

After creating complete stub we will duplicate it to a few different programming and natural languages.

Waiting for your comments (it's low priority question).
@HowProgrammingWorks/contributors

**Comment by belochub** [comment]
@tshemsedinov, Go already has complete page concerning this topic in documentation: [Effective Go](https://golang.org/doc/effective_go.html).
I guess we can just provide a link to it, there's no need for the whole document which will duplicate it.

**Comment by tshemsedinov** [comment]
@belochub Yes, if Go have single place for code conventions, no need to duplicate it, please add link
